lxc/linuxcontainers.org
# Vanilla 1.8

## Done
Updated Vanilla Framework to 1.8 and updated the markup/classes where necessary.

## QA
Run the site locally and click through the pages. 
Check functionality at all viewport sizes (especially the updated nav component)
